modxcms/revolution
# Fix tab tracking error in package manager

### What does it do?
Makes conditionals more precise and error-proof in `beforetabchange` listener.

### Why is it needed?
A new listener method added in #16089 was interfering with package installation and upgrades. Type errors were preventing these processes from completing.

### How to test
Install one new Extra and update one Extra to verify no errors occur and installations are successful.

### Related issue(s)/PR(s)
Resolves #16357
